shogun-lib/src/main/java/de/terrestris/shogun/lib/model/Layer.java

```diff
@@ -83,7 +83,7 @@ public class Layer extends BaseEntity {
         description = "Custom features for the layers that aren't available in the datasource. This might be used " +
             "for custom draw layers or similiar. It's advised to store the features using the GeoJSON format."
     )
-    private GeoJsonObject features;
+    private LayerFeaturesConfig features;
```

Comment:
Do we really need the custom type? I would assume the `GeoJsonObject` contains all relevent GeoJSON type informations?

Reply:
You are right. Reverted